Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+- *State*   
+   State of the address.    
+
+- *Country*   
+   Country of the address.   
+
+- *Charges amount*   
+   Amount of the fee.    
+
+- *Charges currency*   
+   Currency of the fee.      
+
+- *Charges other*   
+   Other fees.     
+
+- *Reference*   
+   Mostly, depending on the payment service provider, the order number.
+
+- *Intented use*   
+   In general, depending on the payment service provider, a further description of the transaction.
+
+- *Charges intended use*   
+   Description of the fee, for example transaction fee.
+
+- *Booking date*   
+   Date on which the payment was posted.   
+
+- *Due date*  
+   If applicable, date on which the payment is to be collected.
+
+- *Payment method*   
+   The way of payment that the customer has chosen, for example, credit card, PayPal&trade;, direct debit.   
+
+- *Status information*   
+   If applicable, specific information on the status of the transaction.  
+
+- *Internal reference*   
+   Actindo specific reference back to the document in the billing document.
+
+- *Original currency*   
+   Currency in which the product(s) were paid.   
+
+- *Original currency amount*   
+   Invoice amount in the original currency.   
+
+
+## Authorizations
+
+*Payments > Authorizations > Tab LIST > Select an authorization transaction*
+
+![Authorization ID](../../Assets/Screenshots/Payments/Authorizations/AuthorizationAttributes.png "[Authorization ID]")
+
+**Authorization transaction "Authorization ID"**
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+    Click this button to close the *Authorization "Authorization ID"* view and return to the list of authorizations. 
+
+
+
+## Authorizations &ndash; Attributes
+
+*Payments > Authorizations > Tab LIST > Select an authorization transaction > Tab Attributes*   
+
+
+![Attributes](../../Assets/Screenshots/Payments/Authorizations/AuthorizationAttributes.png "[Attributes]")
+
+In the left margin column, all available attribute groups are displayed. Click a group to display the attributes that are assigned to this group on the right side of the *Attributes* tab. If the authorization transaction contains attributes that are not assigned, the *Unassigned group* attribute group is automatically displayed in the left margin column.
+
+- ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders)  
+    Attribute group that contains attribute sub-groups. Click the attribute group or the arrow left to the attribute group to expand the group and to display the attribute sub-groups. The following sub-groups are available:
+    - *[Payment service provider] Incoming Payment Base Data*   
+      This folder contains attributes that are specific for the payment service provider who has sent this authorization ticket.
+    - *Incoming Payment Base Data*   
+      This folder contains general attributes that Actindo needs to query payments data. These data support standard functions of Actindo.  
+    - *Unassigned Group*   
```

Comment:
Das wird oben erwähnt (If the authorization transaction contains attributes that are not assigned, the *Unassigned group* attribute group is automatically displayed in the left margin column.) und es kann sein, dass es nicht immer vorkommt (wenn es keine Unassigned attributes gibt). Ich vermute, die Ordner, die angezeigt werden, sind treiberabhängig. Ich finde an sich nicht schlecht, aber es wäre sinnvoll Stefan zu fragen, was standardmäßig vorkommt. Bisher immer so vague gelassen, da es von der Verbindung/vom Treiber abhängt (auch wie die Ordner heißen).

Reply:
Hatte Stefan schon gefragt